nixos/modules/i18n/input-method/fcitx5.nix

```diff
@@ -149,7 +139,10 @@ in
 
     environment.variables = {
       XMODIFIERS = "@im=fcitx";
-      QT_PLUGIN_PATH = [ "${fcitx5Package}/${pkgs.qt6.qtbase.qtPluginPrefix}" ];
+      QT_PLUGIN_PATH = [
+        "${fcitx5Package}/${pkgs.qt5.qtbase.qtPluginPrefix}"
+        "${fcitx5Package}/${pkgs.qt6.qtbase.qtPluginPrefix}"
+      ];
```

Comment:
This change is unnecessary, qt5 picks up plugins from `/run/current-system`, where fctix5 already resides.